packages/quantic/cypress/e2e/default-2/search-box/search-box-expectations.ts

```diff
@@ -0,0 +1,142 @@
+import {InterceptAliases} from '../../../page-objects/search';
+import {should} from '../../common-selectors';
+import {AriaLiveExpectations} from '../../default-1/aria-live/aria-live-expectations';
+import {SearchBoxSelectors, SearchBoxSelector} from './search-box-selectors';
+
+function searchBoxExpectations(selector: SearchBoxSelector) {
+  return {
+    displayClearRecentQueriesButton: (display: boolean) => {
+      selector
+        .clearRecentQueriesButton()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(
+          `${should(display)} display the 'clear recent queries' button`
+        );
+    },
+    querySuggestionsEquals: (querySuggestions: String[]) => {
+      querySuggestions.forEach((querySuggestion, index) => {
+        selector
+          .querySuggestion(index)
+          .invoke('attr', 'data-rawvalue')
+          .should('equal', querySuggestion)
```

Comment:
the reason we don't relay on the actual text is because the text is displayed using a rich text formatter as it contains some bold parts. 

but I agree that this could be improved for more assurance, I will try to improve it.

Reply:
if you take the .text attribute, I think you should be able to get the text and not the formatting tags.

Reply:
made it work with the `.innerText`👍